keywords.txt

```diff
@@ -183,6 +183,18 @@ D173	LITERAL1
 D174	LITERAL1
 D175	LITERAL1
 DMAX	LITERAL1
+LED2 	LITERAL1
+LED3	LITERAL1
+LED4	LITERAL1
+LED5	LITERAL1
+RGB_RED	LITERAL1
+RGB_GREEN	LITERAL1
+RGB_BLUE	LITERAL1
+JOY_LEFT	LITERAL1
+JOY_DOWN	LITERAL1
+JOY_RIGHT	LITERAL1
+JOY_UP	LITERAL1
```

Comment:
https://arduino.github.io/arduino-cli/library-specification/#keywordstxt-format

> keywords.txt is formatted in four fields which are separated by a single true tab (not spaces)

So it can appears not aligned.